app/javascript/utilities/color/WCAGColorContrast.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Check color contrast according to WCAG 2.0 spec
+ * @see http://www.w3.org/TR/WCAG20-TECHS/G17.html
+ * Based on original implementation
+ * https://github.com/doochik/wcag-color-contrast
```

Comment:
In our ruby server side validation we use a [gem](https://github.com/mkdynamic/wcag_color_contrast) which is based off this [JS implementation](https://github.com/doochik/wcag-color-contrast). However,  this library doesn't have an npm package, hence we've just added it as a utility function. 

For now we just want to replicate the validation being done on the server, and to avoid any small differences in output (especially since the ruby implementation will still be used as a primary validation in other places where the brand color appears) we continue to use this one. 

Since I've kept this as a separate utilty and added a small wrapper on top of it in `contrastValidator`, should we want to use a different library in the future, it will be easy to swap out.